src/eos-install-app-helper-installer.py

```diff
@@ -256,17 +277,27 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('--debug', dest='debug', action='store_true')
     parser.add_argument('--initial-setup', dest='initial_setup', action='store_true')
+    parser.add_argument('--app-name', dest='app_name', help='Human readable app name', type=str, required=True)
+    parser.add_argument('--app-id', dest='app_id', help='Flatpak App ID', type=str, required=True)
+    parser.add_argument('--remote', dest='remote', help='Flatpak Remote', type=str, required=True)
+    parser.add_argument('--old-desktop-file-name', dest='old_desktop_file_name', help='File name for .desktop file to remove', type=str, required=True)
+    parser.add_argument('--new-desktop-file-name', dest='new_desktop_file_name', help='File name for .desktop file to add', type=str, required=True)
+    parser.add_argument('--required-archs', dest='required_archs', default=[], nargs='*', type=str)
 
     parsed_args = parser.parse_args()
 
     if parsed_args.debug:
         logging.root.setLevel(logging.DEBUG)
 
-    app_arch = Flatpak.get_default_arch()
-    if app_arch != 'x86_64':
+    if parsed_args.required_archs and Flatpak.get_default_arch() not in parsed_args.required_archs:
         exit_with_error("Found installation of unsupported architecture: %s", app_arch)
 
-    GoogleChromeInstaller(parsed_args.initial_setup)
+    InstallAppHelperInstaller(parsed_args.app_id,
```

Comment:
Yeah, it is a little clunky. The thing here is that the *installer* is different to the helper, even though the whole thing is now `install-app-helper`. Yikes :/